### ISSUE TITLE
[Bug]: Unable to load dynamic library

### ISSUE THREAD

**Issue by mboisvertdupras** [tursodatabase/turso-client-php#8]
### Description

After going through the steps in the documentation, I'm unable to start/restart my php process because of an error loading the liblibsql_php.so library.

### Minimal Reproduction

composer global require darkterminal/turso-php-installer
turso-php-installer install
valet restart

### Exception or Error

Unable to load dynamic library '/Users/<user>/.turso-client-php/liblibsql_php.so' (tried: /Users/<user>/.turso-client-php/liblibsql_php.so (dlopen(/Users/<user>/.turso-client-php/liblibsql_php.so, 0x0009): symbol not found in flat namespace '_sqlite3_wal_manager')

**Comment by mboisvertdupras** [comment]
It's odd that an extra / is added because this is what I have in my php.ini:
`extension=/Users/user/.turso-client-php/liblibsql_php.so`

And this is the output of the 2 commands:
```sh
❯ ls -la | grep .turso 
drwxr-xr-x    6 user  staff       192 22 Sep 11:24 .turso-client-php

~ 
❯ ls ~/.turso-client-php
liblibsql_php.dylib		liblibsql_php.so		libsql_php_extension.stubs.php
```

**Comment by darkterminal** [comment]
Yes that odd... cz I have never tried it directly with herd. If you can fix that It will awesome!

**Comment by darkterminal** [comment]
New release of `turso-php-installer` will fix this issue. Thank for using our extension and installer.
